index.js

```diff
@@ -111,6 +111,8 @@ async function locatePackages(depth) {
 
 function getRenderer(format) {
   switch (format) {
+    case 'md':
+      return require('./render/markdown.js');
     case 'html':
       return require('./render/html.js');
     case 'console':
```

Comment:
PS: can I remove this case and just leave the default one?

Reply:
I like to leave it there as a tiny piece of documentation for future readers that `console` is both an expected value and the default value.